src/main/java/com/rybka/Application.java

```diff
@@ -1,26 +1,45 @@
 package com.rybka;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rybka.config.PropertyInfo;
 import com.rybka.dao.HibernateDAO;
 import com.rybka.service.BaseCurrencyExchangeConnector;
-import com.rybka.service.ExchangeService;
 import com.rybka.service.ExchangeRateConnector;
+import com.rybka.service.ExchangeService;
+import com.rybka.service.PrimeExchangeRateConnector;
 import com.rybka.view.ExchangeView;
 import coresearch.cvurl.io.request.CVurl;
 
+import java.net.http.HttpClient;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Scanner;
 
 public class Application {
+
     public static void main(String[] args) {
 
-        BaseCurrencyExchangeConnector baseCurrencyExchangeConnector = new ExchangeRateConnector(new CVurl());
+        var reader = new PropertyReader(PropertyInfo.PROPERTY_FILE_NAME).getProperties();
+        var currencyConnectorsMap = buildMap();
```

Comment:
buildMap() bad naming(too abstract). Can't recognize what exact map is building. Choose better name that says what exact map is building this map.

Reply:
fixed